File: Python3/프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.py
def solution(a, b):
    answer = 0
    
    if a * (10 ** len(str(b))) + b > a + b * (10 ** len(str(a))):
        answer = a * (10 ** len(str(b))) + b
    else :
        answer = a + b * (10 ** len(str(a)))
    return answer

File: Swift/프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.swift
import Foundation

func solution(_ a:Int, _ b:Int) -> Int {
    let lenA = String(a).count
    let lenB = String(b).count
    
    let option1 = a * Int(pow(10.0, Double(lenB))) + b
    let option2 = a + b * Int(pow(10.0, Double(lenA)))
    
    if option1 > option2 {
        return option1
    } else {
        return option2
    }
}